ratis-server/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -50,9 +50,21 @@ static void setStorageDir(RaftProperties properties, List<File> storageDir) {
     setFiles(properties::setFiles, STORAGE_DIR_KEY, storageDir);
   }
 
-  String REMOVED_GROUPS_DIR_KEY = STORAGE_DIR_KEY + ".removed.groups.dir";
+  String MOVE_REMOVED_GROUPS_ENABLED_KEY = PREFIX
+      + ".move.removed.groups.enabled";
+  boolean MOVE_REMOVED_GROUPS_ENABLED_DEFAULT = false;
+  static boolean moveRemovedGroupsEnabled(RaftProperties properties) {
+    return getBoolean(properties::getBoolean, MOVE_REMOVED_GROUPS_ENABLED_KEY,
+        MOVE_REMOVED_GROUPS_ENABLED_DEFAULT, getDefaultLog());
+  }
+  static void setMoveRemovedGroupsEnabled(RaftProperties properties,
+      boolean shouldMoveRemovedGroups) {
+    setBoolean(properties::setBoolean, MOVE_REMOVED_GROUPS_ENABLED_KEY,
+        shouldMoveRemovedGroups);
+  }
```

Comment:
I think we should add a field for renameDirectory in GroupRemoveRequestProto. This would be similar to deleteDirectory field. Then we do not need this configuration.

Reply:
Done. I have removed this and added renameDirectory